test/framework/src/main/java/org/elasticsearch/test/rest/RestTestLegacyFeatures.java

```diff
@@ -22,9 +22,25 @@
 public class RestTestLegacyFeatures implements FeatureSpecification {
     public static final NodeFeature FEATURE_STATE_RESET_SUPPORTED = new NodeFeature("system_indices.feature_state_reset_supported");
     public static final NodeFeature SYSTEM_INDICES_REST_ACCESS_ENFORCED = new NodeFeature("system_indices.rest_access_enforced");
+    public static final NodeFeature HIDDEN_INDICES_SUPPORTED = new NodeFeature("indices.hidden_supported");
+    public static final NodeFeature COMPONENT_TEMPLATE_SUPPORTED = new NodeFeature("indices.component_template_supported");
+    public static final NodeFeature DELETE_TEMPLATE_MULTIPLE_NAMES_SUPPORTED = new NodeFeature(
+        "indices.delete_template_multiple_names_supported"
+    );
 
     @Override
     public Map<NodeFeature, Version> getHistoricalFeatures() {
-        return Map.of(FEATURE_STATE_RESET_SUPPORTED, Version.V_7_13_0, SYSTEM_INDICES_REST_ACCESS_ENFORCED, Version.V_8_0_0);
+        return Map.of(
+            FEATURE_STATE_RESET_SUPPORTED,
```

Comment:
Definitely `Map.ofEntries(entry(...))` here for readability

Reply:
Agreed, it's becoming too long and spotless does not help here